src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,56 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  // static array to store all animals created.
+  static alive = [];
+
+  constructor(name, health) {
+    // Set name of the animal
+    this.name = name;
+    // set health to 100 by default if its not privided
+    this.health = health || 100;
+    // Add animal to list alive
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
+    // Use to assing name and health accordingly to parent class.
+    super(name, health);
+    this.name = name;
+    this.health = health || 100;
```

Comment:
Redundant assignment: The 'health' property is already set in the parent class constructor. You don't need to set it again in the Herbivore constructor.